Author: Javi Merino <cibervicho@gmail.com>
Description: Fix python interpreter for hg-ssh
 hg-ssh is installed by dh_install instead of setup.py, so it's
 interpreter is not fixed automatically for us.  System-wide commands
 should use /usr/bin/python as the Debian Python Policy recommends.
Bug-Debian: http://bugs.debian.org/654925
Bug-Ubuntu: https://bugs.launchpad.net/bugs/912625
Forwarded: not-needed

--- a/contrib/hg-ssh
+++ b/contrib/hg-ssh
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 #
 # Copyright 2005-2007 by Intevation GmbH <intevation@intevation.de>
 #
